data-plane/receiver/src/main/java/dev/knative/eventing/kafka/broker/receiver/ReceiverVerticleSupplier.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright © 2018 Knative Authors (knative-dev@googlegroups.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dev.knative.eventing.kafka.broker.receiver;
+
+import dev.knative.eventing.kafka.broker.core.security.AuthProvider;
+import io.micrometer.core.instrument.Counter;
+import io.vertx.core.Verticle;
+import io.vertx.core.http.HttpServerOptions;
+import io.vertx.kafka.client.producer.KafkaProducer;
+
+import java.util.Properties;
+import java.util.function.Supplier;
+
+class ReceiverVerticleSupplier implements Supplier<Verticle> {
```

Comment:
Maybe the name for this is `Factory`?

Reply:
Yeah, I'll do.

Reply:
I like `ReceiverVerticleFactory`

Reply:
Done in https://github.com/knative-sandbox/eventing-kafka-broker/pull/985/commits/21e4a13aaa38390c1fd5cc7ad9667d3c53293874, thanks!